specifyweb/frontend/js_src/lib/components/Forms/BaseResourceView.tsx

```diff
@@ -41,6 +41,8 @@ export type ResourceViewState = {
   readonly specifyNetworkBadge: JSX.Element | undefined;
 };
 
+const appResourceNames = new Set(['SpAppResource', 'SpViewSetObj']);
```

Comment:
could you rename this variable to `tablesToHide` or `tableNamesToHide` etc?
i.e try to make the code answer the "why?" question, not "what?"